weblate/checks/placeholders.py

```diff
@@ -137,7 +139,7 @@ def get_description(self, check_obj):
         if not self.has_value(unit):
             return super().get_description(check_obj)
         regex = self.get_value(unit)
-        return mark_safe(
-            escape(_("Does not match regular expression %s."))
```

Comment:
@nijel How can we avoid making translators have to edit this? Is there a way better than manually editing the `.po` files?

Reply:
There is `scripts/locales-fixup` which might help with this (though it will have to be modified for this use-case).